apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/optionalAssertions.kt

```diff
@@ -19,3 +19,17 @@ import java.util.*
  * @since 0.9.0
  */
 fun <T : Optional<*>> Expect<T>.isEmpty(): Expect<T> = addAssertion(ExpectImpl.optional.isEmpty(this))
+
+/**
+ * Expects that the subject of the assertion (an [Optional]) is present.
+ * and returns an [Expect] for the inner type [E].
+ *
+ * Shortcut for more or less something like `feature(Optional<T>::get) { toBe(true) }`
+ * depends on the underlying implementation though.
+ *
+ * @return The newly created [Expect] if the given assertion is success
+ * @throws AssertionError Might throw an [AssertionError] if the given assertion is not a success.
+ *
+ * @since 0.9.0
+ */
+fun <E, T : Optional<E>> Expect<T>.isPresent(): Expect<E> = ExpectImpl.optional.isPresent(this).getExpectOfFeature()
```

Comment:
don't forget too add an overload which expects an assertionCreator (see resultAssertions in  api.fluent.en_GB.kotlin_1_3)